seqr/management/tests/reload_clinvar_all_variants_tests.py

```diff
@@ -0,0 +1,310 @@
+                ]
+            },
+        )
+
+        seqr_clinvar_grch37_snv_indel_models = ClinvarGRCh37SnvIndel.objects.all()
+        self.assertEqual(seqr_clinvar_grch37_snv_indel_models.count(), 0)
+        clinvar_all_variants_grch37_snv_indel_models = ClinvarAllVariantsGRCh37SnvIndel.objects.all()
+        self.assertEqual(clinvar_all_variants_grch37_snv_indel_models.count(), 1)
+        clinvar_all_variants_mito_models = ClinvarAllVariantsMito.objects.all()
+        self.assertEqual(
+            model_to_dict(clinvar_all_variants_mito_models.first()),
+            {
+                'allele_id': 20642,
+                'assertions': ['low_penetrance'],
+                'conditions': [],
+                'conflicting_pathogenicities': None,
+                'gold_stars': None,
+                'pathogenicity': 'Pathogenic/Likely_pathogenic/Established_risk_allele',
+                'submitters': [],
+                'variant_id': 'M-123-G-A',
+                'version': datetime.date(2025, 6, 30)
+            }
+        )
+
+        # Version in Postgres.
+        dv = DataVersions.objects.get(data_model_name='Clinvar')
+        self.assertEqual(dv.version, '2025-06-30')
+        mock_safe_post_to_slack.assert_called_with(
+            SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+            'Successfully updated Clinvar ClickHouse tables to 2025-06-30.',
+        )
+
+
+    @responses.activate
+    def test_batching(self, mock_logger, mock_safe_post_to_slack):
+        # Dynamically build many variants
+        data = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><ClinVarVariationRelease xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/ClinVar_VCV_2.4.xsd" ReleaseDate="2025-06-30">'
+        for i in range(BATCH_SIZE * 2 + 10):
+            data += f'''
+            <VariationArchive VariationID="5603" VariationName="NM_007194.4(CHEK2):c.1283C>T (p.Ser428Phe)" VariationType="single nucleotide variant" Accession="VCV000005603" Version="104" RecordType="classified" NumberOfSubmissions="38" NumberOfSubmitters="36" DateLastUpdated="2025-06-29" DateCreated="2016-03-20" MostRecentSubmission="2025-06-29">
+                <ClassifiedRecord>
+                    <SimpleAllele AlleleID="{i}" VariationID="5603">
+                        <Location>
+                            <SequenceLocation Assembly="GRCh38" Chr="1" variantLength="1" positionVCF="{i}" referenceAlleleVCF="G" alternateAlleleVCF="A"/>
+                        </Location>
+                    </SimpleAllele>
+                    <Classifications>
+                        <GermlineClassification>
+                            <Description>Pathogenic</Description>
+                        </GermlineClassification>
+                    </Classifications>
+                </ClassifiedRecord>
+            </VariationArchive>
+            '''
+        data += '</ClinVarVariationRelease>'
```

Comment:
I think it would be slightly more readable if you defined this as a top level constant and just referenced it here as like `BATCHED_XML_DATA`

Reply:
I still feel this way but would not block approval on it